clang/test/CheckedCRewriter/processor.py

```diff
@@ -0,0 +1,187 @@
+import fileinput 
```

Comment:
This is the script that automatically converts the tests? If you used it, did you check that the output actually makes sense? I.e., not just trust the output of the tool to be right, but confirm that it makes sense, by hand?